arch/ARM/ARMMappingInsn.inc

```diff
@@ -154,13 +154,13 @@
 {
 	ARM_BLX, ARM_INS_BLX,
 #ifndef CAPSTONE_DIET
-	{ ARM_REG_PC, 0 }, { ARM_REG_LR, 0 }, { ARM_GRP_CALL, ARM_GRP_V5T, 0 }, 0, 1
+	{ ARM_REG_PC, 0 }, { ARM_REG_LR, 0 }, { ARM_GRP_CALL, ARM_GRP_V5T, 0 }, 1, 1
```

Comment:
You can see that in the struct insn_map: we have a field for direct branch,
another for indirect branch.